frontend/simple/js/state.js

```diff
@@ -287,11 +310,11 @@ export async function latestContractState (contractId: string) {
     return (Events[e.entry.type] ? Events[e.entry.type].fromObject(e.entry, e.hash) : contracts[e.entry.type].fromObject(e.entry, e.hash))
```

Comment:
I think this line can be shortened as per above in `handleEvent` to use `(Events[ type ] || contracts[ type ]).fromObject(entry, hash)` technique